model/log_iterator.go

```diff
@@ -633,23 +640,25 @@ func (r *logIteratorReader) writeToBuffer(data, buffer []byte, n int) int {
 }
 
 type logIteratorTailReader struct {
-	ctx context.Context
-	it  LogIterator
-	n   int
-	r   io.Reader
+	ctx       context.Context
+	it        LogIterator
+	n         int
+	printTime bool
+	r         io.Reader
 }
 
 // NewLogIteratorTailReader returns an io.Reader that reads up to n log lines
 // from the *reversed* log iterator.
-func NewLogIteratorTailReader(ctx context.Context, it LogIterator, n int) io.Reader {
+func NewLogIteratorTailReader(ctx context.Context, it LogIterator, n int, printTime bool) io.Reader {
```

Comment:
I'd leave the implementation as it is, but avoid having the bool arguments and just make different constructors. 

Alternatly we could have one constructor for both implementations and have a log-iterator-args struct.